js/spa/src/Index.svelte

```diff
@@ -498,6 +498,16 @@
 				composed: true
 			})
 		);
+
+		if (config.js) {
```

Comment:
I would do this onMount, I don't think `render_complete` is actually set by Blocks.svelte